### DISCUSSION TITLE
How can I close the lazygit window without quiting lazygit so that I can reuse the session when I open it again?

### DISCUSSION THREAD

**Discussion by linsui** [LazyVim/LazyVim#4138] [Q&A] [answered]
I use toggleterm to toggle lazygit on and off. So I can always enter the same lazygit session. I recently fount that I can't pass ESC to lazygit anymore and lazyvim has great config for lazygit so I want to switch to that. But it seems I can only quit lazyvim instead of hidding the windows.

**Comment by dotfrag** [comment, marked as answer]
After opening Lazygit (`<leader>gg`), press `<c-/>` to close it. Then press `<leader>gg` again to open it. It will restore the session.

**Reply by linsui** [reply]
Great! Thanks!
